libr/anal/reflines.c

```diff
@@ -1,4 +1,4 @@
-/* radare - LGPL - Copyright 2009-2019 - pancake, nibble */
+/* radare - LGPL - Copyright 2009-2020 - pancake, nibble */
```

Comment:
unrelated changes as well.

Reply:
not completely unrelated, the brainfuck plugin now handles this new reflines feature, so this code was updated to show it